internal/commands/root.go

```
@@ -159,18 +161,30 @@ func (o *credentialOptions) CredentialSetOpts(dockerCli command.Cli, credentialS
 	}
 }
 
-type registryOptions struct {
-	insecureRegistries []string
-}
-
-func (o *registryOptions) addFlags(flags *pflag.FlagSet) {
-	flags.StringSliceVar(&o.insecureRegistries, "insecure-registries", nil, "Use HTTP instead of HTTPS when pulling from/pushing to those registries")
-}
-
 type pullOptions struct {
 	pull bool
 }
 
 func (o *pullOptions) addFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.pull, "pull", false, "Pull the bundle")
 }
+
+func insecureRegistries(dockerCli command.Cli) (registries []string) {
+	registries = []string{}
+
+	info, err := dockerCli.Client().Info(context.Background())
+	if err != nil {
+		logrus.Debugf("could not get docker info: %v", err)
```

I think this function should return an error instead of logging it and return.